rosdep/python.yaml

```
@@ -7267,6 +7267,14 @@ python3-pymongo:
   nixos: [python3Packages.pymongo]
   openembedded: [python3-pymongo@meta-python]
   ubuntu: [python3-pymongo]
+python3-pynmea2:
+  debian:
+    pip:
+      packages: [pynmea2]
```

```suggestion
  debian: [python3-nmea2]
```

This package is available in Debian since at least Debian Stretch so a pip key is not required.